packages/@uppy/utils/src/Translator.ts

```diff
@@ -32,14 +46,16 @@ function insertReplacement (source, rx, replacement) {
  * @license https://github.com/airbnb/polyglot.js/blob/master/LICENSE
  * taken from https://github.com/airbnb/polyglot.js/blob/master/lib/polyglot.js#L299
  *
- * @param {string} phrase that needs interpolation, with placeholders
- * @param {object} options with values that will be used to replace placeholders
- * @returns {any[]} interpolated
+ * @param phrase that needs interpolation, with placeholders
+ * @param options with values that will be used to replace placeholders
  */
-function interpolate (phrase, options) {
+function interpolate(
+  phrase: string,
+  options?: Options,
+): Array<string | unknown> {
   const dollarRegex = /\$/g
   const dollarBillsYall = '$$$$'
-  let interpolated = [phrase]
+  let interpolated: Array<string | unknown> = [phrase]
```

Comment:
why `unknown` if `phrase` is always `string`?

Reply:
Notice the `let` instead of `const`, the variable is overridden a few lines below